examples/bind-and-convert/package.json

```diff
@@ -2,15 +2,20 @@
   "name": "bind-and-convert",
   "version": "3.1.0",
   "dependencies": {
-    "reflect-metadata": "^0.1.13",
-    "tabris": "^3.0.0",
-    "tabris-decorators": "^3.0.0",
+    "reflect-metadata": "^0.1.13"
+  },
+  "optionalDependencies": {
```

Comment:
The app does not provide an alternative if those fail to install; what about keeping those in "dependencies" and using a version range like "^3.3.0-dev"? This will install the nightly and prefer the stable version when it becomes available.